Lib/ImageND/imagend.pd

```diff
@@ -127,19 +127,19 @@ EOD
 # Custom Perl wrapper
 
 sub PDL::convolve{
-    my($a,$b,$c) = @_;
+    my($x,$y,$c) = @_;
     barf("Usage: convolve(a(*), b(*), [o]c(*)") if $#_<1 || $#_>2;
```

Comment:
should this usage error be changed too?

Reply:
My thought was not, since that's still correct in PDL-land (not Perl-land, which is what I was changing)?